contents/blog/aruba-hackathon.md

```diff
@@ -0,0 +1,177 @@
+
+We're constantly building new features, but we have to invite users personally to try them out. It's inefficient and not cool. James, Michael and Annika thought it would be better if PostHog users could join PostHog betas themselves. So, they built it. That is cool.
+
+// TODO: Screenshot of feature preview
+
+Feature previews is pretty simple. It's a list of features we're testing, including screenshots and basic information, that users can simply enable or disable whenever they like. When they do, they're either automatically added to removed from the relevant feature flag.
+
+In future, we want to build this out so users can register their interest in features that aren't ready yet, so they get automatically added to the preview when it goes live.
+
+We hope to roll this out as a usable feature soon.
+
+## Summer social events
+
+- **Team Plops (People & Ops):** Coua Phang and Kendal Hall
+
+Our hackathon isn't just for engineering. The People & Ops team used the time to plan summer events (IRL and virtual) for the PostHog team.
+
+In June they've planned a virtual engraving workshop, in July a virtual escape room experience, and in August we're getting together in Cambridge, UK for a scavenger hunt around the city, and a barbecue.
+
+## Event-based automations
+
+- **Team Automations:** Luke Harries, Ben White, Thomas Obermüller, Cory Watilo 
+
+Luke and Thomas built an MVP for automations in PostHog. In the MVP, automations have a source (event, action or cron job), logic (pause for / pause until), and sources (e.g. send a Slack message, create a GitHub Issue, add to cohort, add to feature flag, send an in-app message etc.).
+
+// TODO: Screenshot of Automation creation
+
+Even this basic functionality creates numerous helpful use cases, but long-term we expect to add more sources and destinations, and the ability to create automations based on event thresholds.
+
+## HouseWatch: Centralized monitoring and management for ClickHouse
+
+- **Team HouseWatch:** Li Yi Yu, Yakko Majuri, and Tim Glaser
+
+PostHog uses ClickHouse as our main event database, but we end up using a huge range of tools (Grafana, Metabase, pganalyze etc.) to monitor and manage it. Team HouseWatch built a centralized dashboard, so we can eliminate all this bloat and have everything in one place.
+
+// TODO: Screenshot of House Watch
+
+The homepage allows us to monitor things like execution count (queries per hour), and memory usage. There's also a slow queries view, which will allow the team to proactively identify problematic queries and reach out to customers to help.
+
+Other features include:
+
+- Custer performance overview
+- Table size visualizations (including size by columns)
+- Ability to kill queries
+- Productized async migrations
+
+Needless to say, this is all backend work you will never see, but House Watch will make it easier for us to keep PostHog fast and reliable for you.
```

Comment:
This is an talent brand piece, ultimately. Worth ending with a shout-out to join the team?